sw/device/lib/dif/dif_entropy.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Entropy Source DIF Checklist"
+---
+
+<!--
+NOTE: This is a template checklist document that is required to be copied over
+to `sw/device/lib/dif/dif_entropy.md` for a new DIF that transitions
+from L0 (Specification) to L1 (Development) stage, and updated as needed.
+Once done, please remove this comment before checking it in.
+-->
+This checklist is for [Development Stage]({{< relref "/doc/project/development_stages.md" >}}) transitions for the [Entropy Source DIF]({{< relref "hw/ip/entropy_src/doc" >}}).
+All checklist items refer to the content in the [Checklist]({{< relref "/doc/project/checklist.md" >}}).
+
+
+
+Type           | Item                 | Resolution  | Note/Collaterals
+---------------|----------------------|-------------|------------------
+Implementation | [DIF_EXISTS][]       | Not Started |
```

Comment:
I think from previous reviews it is okay to mark `DIF_EXISTS` as `Done`.

Reply:
Added WIP :)